test/OpenSpaceToolkit/Mathematics/Solver/NumericalSolver.test.cpp

```diff
@@ -421,16 +421,16 @@ TEST_P(OpenSpaceToolkit_Mathematics_Solver_NumericalSolver_Parametrized, Integra
         NumericalSolver numericalSolver_1 = {
             NumericalSolver::LogType::LogAdaptive,
             std::get<0>(parameters),
-            1e-3,
-            1.0e-15,
-            1.0e-15,
+            1e-1,
```

Comment:
I don't think changing the initial step size does anything here

Reply:
It will for the RK4 I believe, as this test is parametrized.